scripts/flashing/efr32_firmware_utils.py

```diff
@@ -138,6 +138,15 @@ def flash(self, image):
             ['flash', self.DEVICE_ARGUMENTS, image],
             name='Flash')
 
+    def gbl_create(self, argv):
+        """Create gbl file"""
+        self.parser.description = 'Generate a flashing script.'
```

Comment:
Is "Generate a flashing script." the correct description here?

Reply:
Corrected this to "Generate a gbl file"